File: FrontendTemplateProject-main/src/app/models/UserStoryModel/functionalityDto.ts
import { Guid } from 'guid-typescript';

export class FunctionalityDto {
    
    functionalityId!: Guid;
    textFunctionality!: string;
    userStoryRootId!: Guid;
    
    constructor(
        functionalityId: Guid,
        textFunctionality: string,
        userStoryRootId: Guid
    ) {
        this.functionalityId = functionalityId;
        this.textFunctionality = textFunctionality;
        this.userStoryRootId = userStoryRootId;
    }
    
}

File: Sprint/Sprint/Models/FunctionalityDTO.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace Sprint.Models
{
    public class FunctionalityDTO
    {
        
        public Guid FunctionalityId { get; set; }
        public string TextFunctionality { get; set; }

        public Guid UserStoryRootId { get; set; }
    }
}
